src/mappings/DAOTracker/mapping.ts

```diff
@@ -12,8 +12,8 @@ import {
   AvatarContract,
```

Comment:
doa tracker should be removed.
we should track new daos  according to the daoFactory events.

Reply:
But DAOTracker is part of DAOFactory, DAOFactory uses it and accept it as an initialize parameter.

Reply:
will remove it  from daoFactory